Experiments/Experiments/ABTest.swift

```diff
@@ -52,14 +52,16 @@ public extension ABTest {
         let experiments = ABTest.genuineCases.filter { $0.context == context }
 
         await withCheckedContinuation { continuation in
-            guard !experiments.isEmpty else {
+            guard !experiments.isEmpty, ExPlat.shared != nil else {
                 return continuation.resume(returning: ())
             }
 
             let experimentNames = experiments.map { $0.rawValue }
             ExPlat.shared?.register(experiments: experimentNames)
 
+            NotificationCenter.default.post(name: .fetchExperimentAssignments, object: "started")
```

Comment:
Rather than posting the notifications here with untyped objects, can we use the typed invocation on `WooAnalytics.refreshUserData()` where the `ABTest.start()` method is called?